workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectWorldCommandSystem.cs

```diff
@@ -65,8 +64,9 @@ protected override void OnUpdate()
 
                 foreach (var receivedResponse in reserveEntityIdsResponseData.CommandResponses[i].Responses)
                 {
-                    foreach (var worldCommandHandler in worldCommandResponseHandlers)
+                    foreach (var injectable in worldCommandResponseHandlers)
                     {
+                        var worldCommandHandler = (WorldCommands.Requirables.WorldCommandResponseHandler) injectable;
```

Comment:
There is a trick we can do to remove these casts. `IEnumerable` is `covariant`, so internally we can store these as `IEnumerable<IInjectable>` then have the `TryGet<>` function cast it back to `List<Concrete`>.

Reply:
Covariance means you can cast the `IEnumerable<IInjectable>` to the concrete list directly. If we are iterating over the erased types we can cast directly to `List<IInjectable>`.

Reply:
This isn't too far removed from just casting it to object though at that point. Although might be slightly more readable.

Reply:
Arguably it's dangerous to do things that allow you to cast to the wrong type but we have to do the cast somewhere anyway so I don't think we are actually risking anything.